manifests/implementation/elastic/elasticsearch/install.yaml

```diff
@@ -213,3 +213,21 @@ spec:
               image: bitnami/kubectl:1.18
               command: [sh, -c]
               args: ["kubectl create -f {{inputs.artifacts.manifest.path}} --output=yaml > {{outputs.artifacts.manifest.path}}"]
+
+          - name: prepare-parameters
```

Comment:
I thing this template should be used somewhere in the main workflow

Reply:
Removed, when there are only 2 sources it's less code to fill it in two steps.